docs/index.rst

```diff
@@ -0,0 +1,38 @@
+.. meta::
+    :description: HSA runtime implementation
+    :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _index:
+
+=====================
+ROCR documentation
+=====================
+
+The ROCm runtime (ROCR) is AMD's implementation of HSA runtime, which is a thin, user-mode API that exposes the necessary interfaces to access and interact with graphics hardware driven by the AMDGPU driver set and the ROCK kernel driver. To learn more, see :ref:`what-is-rocr-runtime`
+
+You can access ROCR code on our `GitHub repository <https://github.com/ROCm/ROCR-Runtime>`_.
+
+The documentation is structured as follows:
+
+.. grid:: 2
+  :gutter: 3
+
+  .. grid-item-card:: Install
+
+    * :ref:`installation`
+    
+  .. grid-item-card:: API reference
+
+    * :ref:`c-interface-adaptors`
+    * :ref:`environment-variables`
+    * :ref:`api`
+
+  .. grid-item-card:: Contribution
```

Comment:
I don't believe this is a standard box, and it should be removed. Here and from the TOC

Reply:
@lpaoletti We have contribution in many other components as well. Refer MIGraphX, hipTensor, hipBLASLt to name a few. It is also required as a lot of information applies to contributors and not to the users. It is better to segregate information according to the type of audience.

Reply:
Can we change "Contribution" to "How to" instead?  Same with card in index.rst?